src/smt/optimization_solver.h

```diff
@@ -169,20 +174,23 @@ class OptimizationSolver
   }
   ~OptimizationSolver() = default;
 
-  /** Runs the optimization loop for the pushed objective **/
+  /** Runs the optimization loop for the pushed objective 
+   * FIXME: this function currently supports only single objective 
+   * for multiple pushed objectives it always optimizes the first one. 
+   * Add support for multi-obj later **/
```

Comment:
Thanks for adding this comment.
If you use FIXME, then an issue should be created:
https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#todofixme-comments

In this particular case I don't think all of this is needed, so please just remove the `FIXME:` prefix.

Reply:
I can fix this. 
But notice that there's a FIXME like this in NodeValue::inc() in node_value.h . 
You might want to fix that as well.

Reply:
I think a guideline like this is basically preventing people from adding FIXME or TODO, even though they should be. 
This is not a good practice. and I saw FIXMEs in NodeTemplate that are not referencing anything, and I just learned from that. 
Anyways, I will remove this.

Reply:
@barrettcw

Reply:
Fixed

Reply:
The original google guideline is not that restrictive: 
https://google.github.io/styleguide/cppguide.html#TODO_Comments 

> TODOs should include the string TODO in all caps, followed by the name, e-mail address, bug ID, or other identifier of the person or issue with the best context about the problem referenced by the TODO.

Reply:
The FIXMEs without an associated issue are very old and the guidelines were introduced afterwards. So all FIXME since we introduced the dev guidelines have associated issues. The reason why we made the guidelines more restrictive is to avoid the problem of orphaned FIXMEs/TODOs.